src/core.js

```diff
@@ -2,14 +2,15 @@ var QUnit,
 	config,
 	onErrorFnPrev,
 	loggingCallbacks = {},
-	fileName = ( sourceFromStacktrace( 0 ) || "" ).replace( /(:\d+)+\)?/, "" ).replace( /.+\//, "" ),
+	fileName = ( sourceFromStacktrace( 0 ) || "" ).replace( /(:\d+)+\)?/, "" ).replace( /.+\//,
```

Comment:
There are lots of whitespace or linebreak changes like this. They don't belong in this PR, most of them seem just plain wrong. Can you address that? We can then do a proper review (more timely, hopefully).